### ISSUE TITLE
Poor convergence with gradient accumulation

### ISSUE THREAD

**Issue by andreped** [andreped/GradientAccumulator#22]
It was observed that training a model with high class imbalance resulted in poor convergence.

The model did not converge at all. The model was trained with batch size 2, accum steps 8, and tested learning rates 1e-3 and 5e-4 with Adam optimizer.

**Comment by andreped** [comment]
This was due to batch normalization. GA is simply not compatible with BN.
